test/controllers/IndexControllerSpec.scala

```diff
@@ -298,10 +182,10 @@ class IndexControllerSpec extends SpecBase with MockitoSugar with BeforeAndAfter
       verify(mockUserAnswerService, times(1)).set(any())(any(), any())
     }
 
-    "For keepalive no UserAnswer" in {
+    "for keepalive no UserAnswer" in {
```

Comment:
This should also follow the when -> must -> in. Currently it is when -> in which doesn't read well.